cloud-image-val.py

```diff
@@ -29,6 +31,8 @@
 if __name__ == '__main__':
     args = parser.parse_args()
 
+    os.environ['PYTHONPATH'] = ':'.join([f'{os.path.dirname(__file__)}', os.environ['PYTHONPATH']])
```

Comment:
Did this solve the import issue? Maybe it would be nice adding a comment or mention the issue on the commit message